lib/matplotlib/pyplot.py

```diff
@@ -2785,7 +2791,8 @@ def contour(*args, **kwargs):
         ret = ax.contour(*args, **kwargs)
     finally:
         ax._hold = washold
-    if ret._A is not None: sci(ret)
+    if ret._A is not None:
```

Comment:
This needs to be fixed in the `boiler_plate.py` which generates this file (or the fixes will be discarded next time we run it).